index.html

```diff
@@ -57,6 +57,308 @@ <h2>I'm a Software Developer</h2>
+          <div class="project-title">
+            <h4>Profesional Art Printing Data</h4>
+          </div>
+          <div class="support-text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+          </div>
+          <div class="pro-language">
+            <ul>
+              <li><a href="#">HTML</a></li>
+              <li><a href="#">Bootstrap</a></li>
+              <li><a href="#">Ruby</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="pro-btn">
+          <span><a href="#">See project</a></span>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="about">
+        <div class="about-text">
+          <h4>About me</h4>
+        </div>
+        <div class="about-description">
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+        </div>
+        <div class="about-footer">
+          <div class="about-btn">
+            <span><a href="#">Get my resume</a></span>
+          </div>
+          <div class="about-border"></div>
+        </div>
+      </div>
     </section>
+    <section>
+      <div class="language-sec">
+        <div class="lang-container">
+          <div class="lang-img">
+            <img src="./pics/rec.svg" alt="rec" />
+            <h4>Language</h4>
+          </div>
+          <div>
+            <ul class="lang-list">
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">Ruby</a></li>
+              <li><a href="#">HTML</a></li>
+              <li><a href="#">CSS</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="lang-container">
+          <div class="lang-img">
+            <img src="./pics/rec2.svg" alt="rec" />
+            <h4>Frameworks</h4>
+          </div>
+          <div>
+            <ul class="lang-list">
+              <li><a href="#">React.js</a></li>
+              <li><a href="#">Ruby on Rails</a></li>
+              <li><a href="#">RSpec</a></li>
+              <li><a href="#">Capybara</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="lang-container">
+          <div class="lang-img">
+            <img src="./pics/rec3.svg" alt="rec" />
+            <h4>skills</h4>
+          </div>
+          <div>
+            <ul class="lang-list">
+              <li><a href="#">Database Management</a></li>
+              <li><a href="#">Version Control</a></li>
+              <li><a href="#">CLI</a></li>
+              <li><a href="#">API Design</a></li>
+              <li><a href="#">Web Design</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Good job 👏 separating each category with a `div` element but please make sure to use ul and li for categories in the "About myself" section. "Languages", "Frameworks", and "Skills" should be list items, as specified in the [requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)
    - So instead of having separate `div` element for each category replace it with the `li` tag and place all the `li` tags in a `ul` tag as a container. This will ensure you're following the proper semantics as well as giving your page a better chance when it comes to search engine optimization and other supportive devices like screen readers. ✔

Reply:
done :heavy_check_mark: